src/multi-selector.js

```diff
@@ -332,11 +332,13 @@ export default class MultiSelector extends React.Component {
 
   selectItem = (item, e) => {
     let selectedItems = this.state.selectedItems;
-
-    if (includes(selectedItems, item)) {
+    if (this.ref) {
+      this.ref.value = "";
```

Comment:
should we just make the input a [controlled component](https://reactjs.org/docs/forms.html#controlled-components) instead of using a ref? It already has as an onChange, so it's halfway there.

The disadvantage that I see to resetting the input value to empty string this way is that I don't think it will trigger the logic inside of `handleChange` which means the input might have the wrong validation/disabled state on it.